ipaserver/install/dsinstance.py

```diff
@@ -584,9 +584,20 @@ def __create_instance(self):
         # Get the instance ....
 
         inst = DirSrv(verbose=True, external_log=logger)
-        inst.remote_simple_allocate(
-            ldapuri=ipaldap.get_ldap_uri(self.fqdn),
-            password=self.dm_password)
+        inst.local_simple_allocate(
+            serverid=self.serverid,
+            ldapuri=ipaldap.get_ldap_uri(realm=self.realm, protocol='ldapi'),
+            password=self.dm_password
+        )
+
+        # local_simple_allocate() configures LDAPI but doesn't set up the
+        # DirSrv object to use LDAPI.
+        inst.ldapi_enabled = 'on'
+        inst.ldapi_socket = paths.SLAPD_INSTANCE_SOCKET_TEMPLATE % (
+            self.serverid
+        )
+        inst.ldapi_autobind = 'off'
```

Comment:
autobind is enabled in the next installation step. Is there a reason to start with it disabled?

Reply:
This line doesn't change the setting in LDAP. It merely tells the 389-DS installer that ldap_autobind is not available yet and that it has to use password bind. I'll add a comment.